src/platform/forms-system/src/js/definitions/content/directDeposit.jsx

```diff
@@ -74,7 +71,7 @@ export const bankInfoHelpText = (
       <strong>Note:</strong> The Department of the Treasury requires us to make
       electronic payments. If you don’t want to use direct deposit, you’ll need
       to call the Department of the Treasury at{' '}
-      <Telephone contact={CONTACTS.TESC} />. Ask to talk with a representative
+      <va-telephone contact="8882242950" />. Ask to talk with a representative
```

Comment:
The web component does not keep a directory of numbers like the React component did. This value is the TESC phone number taken from the [web component's Storybook page](https://design.va.gov/storybook/?path=/docs/components-va-telephone--default): 

![Screenshot 2023-04-21 at 2 58 16 PM](https://user-images.githubusercontent.com/872479/233724268-8ea980d3-638a-409c-9596-ebec7cee0ad8.png)